lmp/model/_transformer.py

```diff
@@ -0,0 +1,199 @@
+import math
+import torch
+import numpy as np
+from torch.utils.tensorboard import SummaryWriter
+from torch.nn import Linear
+from torch.nn.functional import softmax, relu
+
+
+class PositionalEncoding(torch.nn.Module):
+    "Implement the PE function."
```

Comment:
Always use `r"""` and `"""` to enclose docstring.
Specify which formula you use to implement positional encoding in docstring.

Reply:
Add type check and value check.
Do follow google python style guide.
reference: https://google.github.io/styleguide/pyguide.html